sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_configure.py

```diff
@@ -68,6 +69,8 @@ def configure_azure_monitor(**kwargs) -> None:
      `{"azure_sdk": {"enabled": False}, "flask": {"enabled": False}, "django": {"enabled": True}}`
      will disable Azure Core Tracing and the Flask instrumentation but leave Django and the other default
      instrumentations enabled.
+    :keyword list[~opentelemetry.sdk.trace.SpanProcessor] span_processors: List of `SpanProcessor`s to
```

Comment:
Apparently, sphinx doesn't like it when you have an inline literal and adjacent text not separated by some whitespace. Specifically, it's complaining about "`SpanProcessor`s".

For this, I think some options are:
```
`SpanProcessor`\s
`SpanProcessor` objects
`SpanProcessors`

```